bridges/RadioMelodieBridge.php

```diff
@@ -13,6 +13,17 @@ public function collectData(){
 		$html = getSimpleHTMLDOM(self::URI . '/actu/')
 			or returnServerError('Could not request Radio Melodie.');
 		$list = $html->find('div[class=displayList]', 0)->children();
+
+		$dateFormat = 'EEEE  d LLLL yyyy à HH:mm';
+		$dateFormatter = new IntlDateFormatter(
```

Comment:
That would require intl extention to be enabled, which is not in minimal requirements of rss-bridge.
Can it be parsed via https://www.php.net/manual/ru/function.date-parse-from-format.php?

Reply:
Hello !

As date_parse_from_format could not handle localized day of week and month names, I switched to strptime which is totally native (but needs to have access to the French locale)/